working-group-abilities.md

```diff
@@ -0,0 +1,37 @@
+# Working Group Abilities
+
+Established working groups in the OSSF are given broad latitude to accomplish their goals.
+The TAC aims to only require formal approval in limited cases.
+This document outlines some of the explicitly permitted activities and resources made
+available to working groups.
+It is not intended to be exhaustive, and will grow over time.
+
+When in doubt, reach out to the TAC with any questions!
+
+## Repositories
+
+We expect that working groups will need to create repositories for code, tools, and other projects.
+These should be created in the [ossf](github.com/ossf) organization where possible to simplify
+management, but working groups can also create new organizations where necessary.
+
+If any of these projects grow large enough, they can be "spun-off" into separate working groups by
+the TAC.
+As a general rule-of-thumb, if a sub-project requires its own standing meeting, it might be time to
+spin-it-off.
```

Comment:
Can we have some lightweight process for WGs to request approval for creation of new projects?  This is for visibility and alignment across working groups (which might be contemplating similar projects). Perhaps the process is just sending an email to the TAC (and WG Leads?) and asking for 2 individuals to give a 'thumbs up'.  Something for the TAC to discuss.  This aim is not to hamper creativity or progress (both are loved in the OpenSSF!), but to provide ample opportunity for collaboration.

Reply:
I'd prefer to leave this up to Working Groups to do on their own. A "firehose" list to announce these too would be fine with me, but working groups need to be able to create repos to do their work.

Reply:
I'm not sure if Projects and repos are conflated in this comment thread.

There should be one process for Technical Initiatives (Projects and Working Groups) which goes through the TAC. They might be recommended by a Working Group but only the TAC approves through the established process.

The use of the word "projects" here...

> If any of these projects grow large enough, they can be "spun-off" into separate working groups by
> the TAC.

... might be confusing. Can we replace that with "efforts" or some other word that isn't as overloaded as project/Project?

Reserved words: Project, Working Group, Technical Initiative  :)

Reply:
How about adding something like we said for papers, e.g. "For new efforts, WGs are encouraged to send an email to openssf-tac@lists.openssf.org describing the motivation and goals. This increases opportunities for collaboration, visibility and feedback across the broader OpenSSF community."